packages/astro/components/Shiki.js

```diff
@@ -15,14 +15,14 @@ function stringify(opts) {
 
 /**
  * @param {import('shiki').HighlighterOptions} opts
- * @returns {Promise<import('shiki').Highlighter>}
+ * @returns {Promise<import('shiki').HighlighterOptions>}
  */
-async function resolveHighlighter(opts) {
+export async function resolveHighlighterOptions(opts) {
```

Comment:
So the `export` here is the fix?

Reply:
No, the order swapping is the fix: https://github.com/withastro/astro/pull/6198/files#diff-766f75438aeb2486b5bf2c8f6f3a6521fb48e42b6b51b9be6077cdad5f85c162R22-R25

the export is to unit test this function.

Reply:
Ahhh OK thanks!